app/cdap/components/PipelineList/DraftPipelineView/DraftActions/index.tsx

```diff
@@ -46,25 +48,39 @@ class DraftActions extends React.PureComponent<IProps, IState> {
 
   private handlePipelineExport = () => {
     const draft = this.props.draft;
-
     this.pipelineConfig = {
       name: draft.name,
       description: draft.description,
       artifact: draft.artifact,
       config: draft.config,
     };
 
-    if (window.Cypress) {
-      this.openExportModal();
-      return;
-    }
-    const postExportCb = () => {
-      this.pipelineConfig = {};
-      this.setState({ showPopover: false });
+    const params = {
+      context: getCurrentNamespace(),
+      draftId: draft.id,
     };
-
-    // Unless we are running an e2e test, just export the pipeline JSON
-    downloadFile(this.pipelineConfig, postExportCb);
```

Comment:
Isn't this calling to export the draft configuration?

However the config and description is somehow missing

Reply:
I moved it to the subscribe complete